lib/index.js

```diff
@@ -178,6 +178,34 @@ function _ensure_cheerio(value) {
     return value;
 }
 
+class DictionaryAsset extends BaseAsset {
+    constructor() {
+        super();
+        this._object_type = "DictionaryObject";
+        this._content_type = "text/html";
```

Comment:
I don't think this is actually correct, since afaik there's no HTML involved here. To be honest I'm not sure what the content type of a "word" is